go-app.js

```diff
@@ -46,6 +48,27 @@ go.utils = {
         return ('' + sum).slice(-1) == check;
     },
 
+    now: function () {
+        return moment();
+    },
+
+    generate_case_number: function (im, facility_code) {
+        return im.api_request('kv.incr', {
+            key: 'facility_code:' + facility_code,
+            amount: 1,
+        })
+        .then(function(result){
+            return result.value;
+        })
+        .then(function(sequence_number) {
+            return [
+                go.utils.now().format('YYYYMMDD'),
+                facility_code,
+                sequence_number,
+            ].join('-');
+        });
+    },
```

Comment:
Where did all of this code come from? I thought go-app.js was the file generated by grunt but I don't see this stuff in src/app.js below?